src/main/java/hudson/os/windows/ManagedWindowsServiceLauncher.java

```diff
@@ -453,7 +454,7 @@ protected String determineHost(Computer c) {
     
     private String createAndCopyJenkinsSlaveXml(String java, String serviceId, PrintStream logger, SmbFile remoteRoot) throws IOException {
         logger.println(Messages.ManagedWindowsServiceLauncher_CopyingSlaveXml(getTimestamp()));
-        String xml = generateSlaveXml(serviceId,
+        String xml = generateSlaveXml(getClass(), serviceId,
```

Comment:
Better style to use `ManagedWindowsServiceLauncher.class`, unless the class is `final`, or you really expect to vary the resource according to the package of a subclass.